content/token-program.md

```diff
@@ -430,24 +455,22 @@ dist/
 Add the following to the `scripts` object in `package.json`:
 
 ```json
-"start": "tsc && node dist/index.js"
+"start": "ts-node src/index.ts"
 ```
 
 Add your Keypair to the `.env` file as an environment variable called, `PRIVATE_KEY`. If you need to generate a new Keypair, please refer to Module 1 Lesson 2 for how to do so. It should look something like this but with different numbers:
```

Comment:
remove last sentence since reference deleted

Reply:
Good catch. Reworked all of this with the new `npx create-solana-client` command